packit/schema.py

```diff
@@ -396,6 +396,7 @@ class CommonConfigSchema(Schema):
 
     # Packaging tool used for interaction with lookaside cache
     pkg_tool = fields.String(missing=None)
+    version_update_mask = fields.String(missing=r"\d+\.\d+\.")
```

Comment:
I am wondering about the default, could we by default not check this mask and do that only when a user configures this? This way if I understand it correctly, the Packit's default behaviour will change.

Reply:
We've agreed on the same with Maja in the meantime..;-D

Reply:
oh ok :D nice